packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -113,19 +114,21 @@ export const useArbTokenBridge = (
 
   const { addPendingTransaction } = useTransactionHistory(walletAddress)
 
+  const { destinationAddress } = useDestinationAddressStore()
+
```

Comment:
Adding destination address in useArbTokenBridge fixes the bug of the custom-destination-address's balance not being updated after a transaction was made. We test for this in e2e hence fixing in this branch.
For more context check thread: https://offchainlabs.slack.com/archives/C048JAGDWN9/p1706102964590739